Source/Core/VideoBackends/OGL/TextureConverter.cpp

```diff
@@ -213,9 +213,14 @@ void Shutdown()
 	s_texConvFrameBuffer[1] = 0;
 }
 
+// dstWidth, dstHeight in pixels
+// readStride in bytes, writeStride in bytes
+
+
+template <u32 dst_pixel_size>
 static void EncodeToRamUsingShader(GLuint srcTexture,
```

Comment:
tbh, this function still read from an hardcoded RGBA texture, so making dst_pixel_size optional is confusing for me.
Choosing dst_pixel_size=1 right now means to only read back the first quarter of the pixels.
For our XFB texture, the resolution is only dstWidth/2 x dstHeight. Two YUV "pixels" are already encoded within one RGBA pixel, so which pixel size are you refering to?